lib/clientv2/networkpolicy_e2e_test.go

```diff
@@ -77,20 +77,28 @@ var _ = testutils.E2eDatastoreDescribe("NetworkPolicy tests", testutils.Datastor
 			be.Clean()
 
 			By("Updating the NetworkPolicy before it is created")
+			var rv string
+			if config.Spec.DatastoreType != apiconfig.Kubernetes {
+				rv = "1234"
+			} else {
+				// Resource version for KDD is a combination of both the CRD and K8s NP backed
+				// resources separated by a slash.
+				rv = "1234/1234"
```

Comment:
Should probably use `joinRevision`?

Reply:
we can do - i'll have to make all that public though